contracts/contracts/sbtc-registry.clar

```diff
@@ -14,6 +21,11 @@
 (define-data-var current-signer-set (list 128 (buff 33)) (list))
 (define-data-var current-aggregate-pubkey (buff 33) 0x00)
 (define-data-var current-signer-principal principal tx-sender)
+(define-data-var active-protocol-contracts {governance: principal, deposit: principal, withdrawal: principal} {
```

Comment:
IMO it would be better to have:

```
(define-map active-protocol-contracts (buff 1) principal)
```

Firstly because it's a little more future-proof if we wanted to use this map to store new "roles". We also only ever need to fetch 1 role, too.

Reply:
Updated